src/OrbitBase/Logging.cpp

```diff
@@ -14,6 +14,24 @@
 
 static absl::Mutex log_file_mutex(absl::kConstInit);
 std::ofstream log_file;
+const int kLogFileLifetimeMinutes = 60 * 24 * 7;  // one week
+
+void RemoveOldLogFiles(const std::filesystem::path& log_dir) {
```

Comment:
Maybe return ErrorMessageOr<void> here and have caller desided what to do in case removing old files fails.

Or/and(?) rename this TryRemoveOldLogFiles

Reply:
Thanks for the suggestion. Changed this method as `ErrorMessageOr<void> TryRemoveOldLogFiles(const std::filesystem::path& log_dir)`.